UI/window-basic-settings.cpp

```diff
@@ -291,6 +292,9 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 	ui->setupUi(this);
 
 	setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+	QScreen *screen = App()->GetMainWindow()->windowHandle()->screen();
```

Comment:
What if we put this code inside the windows `#ifdef` since it's windows only bug and we already have a bunch of _WIN32 ifdefs in this method.

Reply:
Confirmed, already works just fine on mac.  Adding ifdefs.